backend/src/models/role.js

```diff
@@ -0,0 +1,16 @@
+import db from '../database/db.js';
+import { Sequelize } from 'sequelize';
+
+function Role(database) {
+  const Model = db(database).define('role', {
+    id: { type: Sequelize.SMALLINT, primaryKey: true },
+    role: Sequelize.STRING
```

Comment:
```suggestion
   name: Sequelize.STRING
```

**NOTA: ** para tablas las cuales sólo llevan un campo de referencia, utilizar `name`

Reply:
La actualización del Script la hice de manera "manual"

Reply:
Te explico de nuevo.
Ese script debe contener el código de las creación de la tabla. Si no, todas las nuevas empresas no vendrán con dicha tabla.